### PR TITLE
Allow admins to read EOD logs

### PR DESCRIPTION
**Pull request by stirlingv**
@tanderegg This will fail the travis build, since it doesn't include all the regression changes included in the travis_timeout branch. It will also likely have merge conflicts in the travis.yml file and the deploy_research_environment.yml (the changes in this are to remove what looks like a typo) files. I should be able to resolve those easily once the travis_timeout branch is merged in.

### CODE REVIEW

File: deploy/roles/eod/tasks/main.yaml [hunk 1]
```diff
@@ -49,10 +49,27 @@
   tags:
     - eod
 
+- name: Create admin group
+  group:
+    name: "{{ restricted_group_name }}"
+    state: present
+
```
Comment (tanderegg) [inline deploy/roles/eod/tasks/main.yaml]: @stirlingv Isn't this group already created in the `common` role?

File: deploy/roles/eod/tasks/main.yaml [hunk 2]
```diff
@@ -49,10 +49,27 @@
...
+    mode: 0640
+
+- name: Set recursive admin permissions on log file
+  acl:
+    path: "{{ eod_install_directory }}/log"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rX
+    default: yes
+    state: present
```
Comment (tanderegg) [inline deploy/roles/eod/tasks/main.yaml:76]: @stirlingv Don't we need to sections here, one with `default: yes` and the other `recursive: yes`?

**Review by tanderegg — CHANGES_REQUESTED** [review]

File: deploy/roles/eod/tasks/main.yaml (hunk 1 above)
Comment (stirlingv) [reply deploy/roles/eod/tasks/main.yaml]: @tanderegg It is, but in travis we don't call the deploy_research host before running the research_eod host. So the common role is not called. 
```
- hosts: research_eod
  become: yes
  become_method: sudo
  roles:
    - eod
    - odbc
```

Maybe we should add a when condition to only add the group when running in travis?

Comment (tanderegg) [reply deploy/roles/eod/tasks/main.yaml]: @stirlingv The common role is applied in the `hosts: research_environment` block at the top of the playbook, which applies to all servers in the RE, so you can safely remove this block, thanks!

File: deploy/roles/eod/tasks/main.yaml (hunk 2 above)
Comment (tanderegg) [reply deploy/roles/eod/tasks/main.yaml:76]: @stirlingv I meant "two sections" not "to sections" :)

Comment (stirlingv) [reply deploy/roles/eod/tasks/main.yaml:76]: @tanderegg I am looking at this now - When I first did this there was a reason I didn't have a recursive parameter. I'll keep ya posted...

**Review by tanderegg — APPROVED** [review]

**Pull request merged**